api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/ResumoMensalQueries.java

```diff
@@ -63,24 +69,40 @@ public static String getPatientsTransferredFromAnotherHealthFacilityDuringTheCur
             + "         ON type.encounter_id = e.encounter_id "
             + "WHERE  p.voided = 0 "
             + "       AND e.voided = 0 "
-            + "       AND e.encounter_type = %d "
+            + "       AND e.encounter_type = ${masterCard} "
             + "       AND e.location_id = :location "
             + "       AND e.encounter_datetime BETWEEN :startDate AND :endDate "
             + "       AND transf.voided = 0 "
-            + "       AND transf.concept_id = %d "
-            + "       AND transf.value_coded = %d "
+            + "       AND transf.concept_id = ${transferFromOther} "
```

Comment:
The specs says **Exclude all patients transferred-in** based on those parameters, looks like we are getting sum of those patients in addition to the lower union. Are we NOt excluding those from the resultant query?

Reply:
@ningosi I think you might be looking at specs for B1, this is B2.